frontend/src/layouts/CampLayout.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-app style="background: #90A4AE">
+  <v-app style="background: #F4F4F4">
```

Comment:
Habe jetzt in meinem Pull das Material Blue-Grey verwendet. #376 2ba69ad

Reply:
Finde ich gut die Material-Farben zu verwenden. Diese Schattierung scheint mir aber immer noch etwas schwer (dunkel). Ich habe das ursprünglich geändert weil mir Leute denen ich die aktuelle Version gezeigt habe das bemängelt haben, und kann es auch selber sehr gut nachvollziehen. Aber kann man ja in Zukunft noch gut dran schrauben.